node/public/script/main.js

```diff
@@ -33,14 +33,14 @@ if (window.location.pathname == "/") {
 
     //on attend le signal init du serveur qui envoie la main et le board
     socket.on('init', function (init) {
-
+        console.log(init.board);
         //on parcours le board pour l'afficher
         for (indexBoard = 0; indexBoard < 4; indexBoard++) {
             for (indexBoard2 = 0; indexBoard2 < 6; indexBoard2++) {
                 let tmp = indexBoard + 1;
                 let tmp2 = indexBoard2 + 1;
                 if (init.board[indexBoard][indexBoard2] != undefined) {
```

Comment:
Code smell. Make sure you understand `undefined` correctly: https://codeburst.io/javascript-null-vs-undefined-20f955215a2